build-apisix-runtime.sh

```diff
@@ -163,6 +173,13 @@ make -j`nproc`
 sudo make install
 cd ..
 
+cd lua-resty-events-${lua_resty_events_ver} || exit 1
+sudo install -d "$OR_PREFIX"/lualib/resty/events/
+sudo install -m 664 lualib/resty/events/*.lua "$OR_PREFIX"/lualib/resty/events/
+sudo install -d "$OR_PREFIX"/lualib/resty/events/compat/
+sudo install -m 644 lualib/resty/events/compat/*.lua "$OR_PREFIX"/lualib/resty/events/compat/
```

Comment:
Why not use `sudo OPENRESTY_PREFIX="$OR_PREFIX" make install` ?

Reply:
There is a problem with the official script. This is the result of debugging.

https://github.com/Kong/lua-resty-events/blob/main/Makefile